openapi.yaml

```diff
@@ -208,10 +441,348 @@ definitions:
             type: string
             format: url
             description: A link to further information about the error
-            example: https://developer.oregonstate.edu/documentation/error-reference#1234
+            example: https://developer.oregonstate.edu/documentation/Error-reference#1234
   Errors:
     properties:
       errors:
         type: array
         items:
           $ref: '#/definitions/Error'
+  GameId:
+    type: string
+    description: A unique ID of a poker game
+  GameIds:
+    type: array
+    items:
+      $ref: '#/definitions/GameId'
+  GamePatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/GameId'
+          type:
+            $ref: '#/definitions/GameType'
+          attributes:
+            type: object
+            properties:
+              round:
+                $ref: '#/definitions/Round'
+              playerIds:
```

Comment:
I think when updating a game, you'll want to specify a list of `memberIds` instead